bugwarrior/docs/services/linear.rst

```diff
@@ -40,6 +40,24 @@ Options
    are processed.
    The default value is ``{{label|replace(' ', '_')}}``.
 
+.. describe:: status_types
+
+   Comma-separated list of Linear status types that should be included. The
+   default statuses for a team and any custom statuses fall into one of a few
+   pre-defined types: `backlog`, `unstarted`, `started`, `completed`, and
+   `canceled`, and are case-sensitive. See
+   https://linear.app/docs/configuring-workflows#overview. The default value is
+   ``backlog,unstarted,started``.
```

Comment:
Using the config directive is especially preferred for structured values since they're formatted pretty differently in toml.

```suggestion
   https://linear.app/docs/configuring-workflows#overview. The default value is:
   
   .. config::
       :fragment: linear

       linear.status_types = backlog,unstarted,started
```